--- tests/Fixtures/Standards/Typo3Update/Sniffs/Classname/UseSniff/InputFileForIssues.php
+++ PHP_CodeSniffer
@@ -19,8 +19,8 @@
  * 02110-1301, USA.
  */

-use \Tx_Extbase_Domain_Model_Backenduser;
-use Tx_Extbase_Domain_Model_Backenduser;
+use \TYPO3\CMS\Extbase\Domain\Model\BackendUser;
+use TYPO3\CMS\Extbase\Domain\Model\BackendUser;
 use TYPO3\CMS\Extbase\Mvc\Cli\Command;
 use \TYPO3\CMS\Extbase\Mvc\Cli\Command;
 use TYPO3\CMS\Perm\Controller\PermissionAjaxController;
