--- tests/Fixtures/Standards/Typo3Update/Sniffs/Classname/TypeHintSniff/InputFileForIssues.php
+++ PHP_CodeSniffer
@@ -25,10 +25,10 @@
 function something(\TYPO3\CMS\Extbase\Domain\Model\BackendUser $user)
 {
 }
-function something(\Tx_Extbase_Domain_Model_Backenduser $user)
+function something(\TYPO3\CMS\Extbase\Domain\Model\BackendUser $user)
 {
 }
-function something(Tx_Extbase_Domain_Model_Backenduser $user)
+function something(\TYPO3\CMS\Extbase\Domain\Model\BackendUser $user)
 {
 }

@@ -40,10 +40,10 @@
     function something(\TYPO3\CMS\Extbase\Domain\Model\BackendUser $user)
     {
     }
-    function something(\Tx_Extbase_Domain_Model_Backenduser $user)
+    function something(\TYPO3\CMS\Extbase\Domain\Model\BackendUser $user)
     {
     }
-    function something(Tx_Extbase_Domain_Model_Backenduser $user)
+    function something(\TYPO3\CMS\Extbase\Domain\Model\BackendUser $user)
     {
     }

@@ -53,10 +53,10 @@
     public function something(\TYPO3\CMS\Extbase\Domain\Model\BackendUser $user)
     {
     }
-    public function something(\Tx_Extbase_Domain_Model_Backenduser $user)
+    public function something(\TYPO3\CMS\Extbase\Domain\Model\BackendUser $user)
     {
     }
-    public function something(Tx_Extbase_Domain_Model_Backenduser $user)
+    public function something(\TYPO3\CMS\Extbase\Domain\Model\BackendUser $user)
     {
     }

